### ISSUE TITLE
Add support for writing to collectd

### ISSUE THREAD

**Issue by mateusz-gozdek-sociomantic** [unixsurfer/haproxystats#11]
It would be nice to have a support  for writing stats to collectd instead of graphite.

[Collectd format](https://collectd.org/wiki/index.php/Plain_text_protocol#PUTVAL) is not so different from graphite, so it shouldn't be hard to do.

General idea about configuration:
```
[collectd]
inverval = 30
socket-path = /var/run/collectd.socket
```

This will requre adding custom types entries to collectd config unless we have it directly in collectd.

**Comment by unixsurfer** [comment]
I am fine with the idea. What do you mean by 'adding custom types entries' ?

BTW: The interval is used for the retry logic, when we fail to send data to an entry point (collectd in this case), and not as a way to set how ofter we fetch data from haproxy.

**Comment by mateusz-gozdek-sociomantic** [comment]
As far as I understand, collectd is sending data packed, not like graphite which supports plain text protocol, so both sender and receiver must know all fields, so adding additional [type](https://collectd.org/documentation/manpages/types.db.5.shtml) would be probably desired.

Something like:
```
haproxy_server    chkfail:GAUGE:0:U, cli_abrt:GAUGE:0:U, econ:GAUGE:0:U<...>
```
so to collectd we sent PUTVAL matching [naming schema](https://collectd.org/wiki/index.php/Naming_schema)
```
PUTVAL "localhost/haproxy-1/haproxy_server interval=30 1179574444:0:0:0<...>
```

About the interval, in my example, I thought about collectd inverval setting, so `interval=30`.

**Comment by unixsurfer** [comment]
Does it mean that for each metric we have to create type entry ? some are gauges and others counters. There are also some metrics, which have a string as a value.

**Comment by mateusz-gozdek-sociomantic** [comment]
No, each group of metric should have it's own type. So probably:
- `haproxy_daemon`
- `haproxy_daemon_avg`
- `haproxy_server`
- `haproxy_server_avg`
- `haproxy_backend`
- `haproxy_backend_avg`
- `haproxy_frontend`

or whatever makes more sense.
